### PR TITLE
Adding all classes conforming to UIAppearance to README

### PR DESCRIPTION
**Pull request by maju6406**

### CODE REVIEW

**Comment by blakewatters** [comment]
General question: How are we going to ensure that this list stays up to date as the codebase evolves?

File: README.md [hunk 1]
```diff
@@ -223,6 +223,67 @@ While the Atlas does provide highly customizable TableView and CollectionView ce
 
 The presenters are `ATLParticipantPresenting`, `ATLConversationPresenting`, and `ATLMessagePresenting`.
 
+## Configuring UI Appearance
+
+Atlas takes advantage of Apple's [UIAppearance](https://developer.apple.com/library/ios/documentation/UIKit/Reference/UIAppearance_Protocol/) protocol which lets you change UI appearance very easily. The follow is a list of all Atlas properties conforming to `UIAppearance`:
```
Comment (blakewatters) [inline README.md]: “The following is a list"

File: README.md [hunk 2]
```diff
@@ -223,6 +223,67 @@ While the Atlas does provide highly customizable TableView and CollectionView ce
...
+
+### ATLMessageCollectionViewCell
+##### (ATLOutgoingMessageCollectionViewCell and ATLIncomingMessageCollectionViewCell extend this class)
+```objc
```
Comment (blakewatters) [inline README.md]: Throw spacing in between the header and the code block

File: README.md [hunk 3]
```diff
@@ -223,6 +223,67 @@ While the Atlas does provide highly customizable TableView and CollectionView ce
...
+@property (nonatomic) UIFont *messageTextFont
+@property (nonatomic) UIColor *messageTextColor
+@property (nonatomic) UIColor *messageLinkTextColor
+@property (nonatomic) UIColor *bubbleViewColor
+@property (nonatomic) CGFloat bubbleViewCornerRadius
+```
+
+### ATLAddressBarTextView
+```objc
```
Comment (blakewatters) [inline README.md]: Throw spacing in between the header and the code block

File: README.md [hunk 4]
```diff
@@ -223,6 +223,67 @@ While the Atlas does provide highly customizable TableView and CollectionView ce
...
+@property (nonatomic) UIFont *addressBarFont
+@property (nonatomic) UIColor *addressBarTextColor
+@property (nonatomic) UIColor *addressBarHighlightColor
+```
+
+### ATLAvatarImageView
+```objc
```
Comment (blakewatters) [inline README.md]: Throw spacing in between the header and the code block

File: README.md [hunk 5]
```diff
@@ -223,6 +223,67 @@ While the Atlas does provide highly customizable TableView and CollectionView ce
...
+@property (nonatomic) CGFloat avatarImageViewDiameter
+@property (nonatomic) UIFont *initialsFont
+@property (nonatomic) UIColor *initialsColor
+@property (nonatomic) UIColor *imageViewBackgroundColor
+```
+
+### ATLConversationCollectionViewHeader
+```objc
```
Comment (blakewatters) [inline README.md]: Throw spacing in between the header and the code block

File: README.md [hunk 6]
```diff
@@ -223,6 +223,67 @@ While the Atlas does provide highly customizable TableView and CollectionView ce
...
+@property (nonatomic) UIFont *participantLabelFont
+@property (nonatomic) UIColor *participantLabelTextColor
+```
+
+### ATLConversationTableViewCell
+```objc
```
Comment (blakewatters) [inline README.md]: Throw spacing in between the header and the code block

File: README.md [hunk 7]
```diff
@@ -223,6 +223,67 @@ While the Atlas does provide highly customizable TableView and CollectionView ce
...
+@property (nonatomic) UIFont *lastMessageLabelFont
+@property (nonatomic) UIColor *lastMessageLabelColor
+@property (nonatomic) UIFont *dateLabelFont
+@property (nonatomic) UIColor *dateLabelColor
+@property (nonatomic) UIColor *unreadMessageIndicatorBackgroundColor
+@property (nonatomic) UIColor *cellBackgroundColor
+```
+
+### ATLParticipantSectionHeaderView
+```objc
```
Comment (blakewatters) [inline README.md]: Throw spacing in between the header and the code block

File: README.md [hunk 8]
```diff
@@ -223,6 +223,67 @@ While the Atlas does provide highly customizable TableView and CollectionView ce
...
+@property (nonatomic) UIFont *sectionHeaderFont
+@property (nonatomic) UIColor *sectionHeaderTextColor
+@property (nonatomic) UIColor *sectionHeaderBackgroundColor
+```
+
+### ATLParticipantTableViewCell
+```objc
```
Comment (blakewatters) [inline README.md]: Throw spacing in between the header and the code block

**Comment by blakewatters** [comment]
Also, what do you think about hot linking all the class names to the CocoaDocs API reference?

**Comment by maju6406** [comment]
Updated content based on your feedback, removed references of ATLParticipantPicker, and added links to all the classes.

**Comment by kcoleman731** [comment]
Thank you @maju6406

**Pull request merged**